src/calculator/votes.ts

```diff
@@ -102,6 +100,31 @@ export async function getVotesWithCoefficients(
   return (await Promise.all(votePromises)).flat();
 }
 
+export function scoreToCoefficient(
```

Comment:
```suggestion
function scoreToCoefficient(
```

it's an internal function right? not seeing usage outside

Reply:
@boudra yes now that it's there it makes sense I forgot it, thank you!